openshift-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/OpenshiftUndeployMojoTest.java

```diff
@@ -0,0 +1,71 @@
+package org.eclipse.jkube.maven.plugin.mojo.develop;
```

Comment:
license header missing, Our license check is not working as expected. I've created #355 to fix it